proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -87,14 +87,6 @@ message SignDocDirectAux {
 
   // sequence is the sequence number of the signing account.
   uint64 sequence = 5;
-
-  // Tip is the optional tip used for transactions fees paid in another denom.
-  // It should be left empty if the signer is not the tipper for this
-  // transaction.
-  //
-  // This field is ignored if the chain didn't enable tips, i.e. didn't add the
-  // `TipDecorator` in its posthandler.
-  Tip tip = 6;
```

Comment:
It would rather see a deprecated since, for simplifying the api module release (as discussed on Slack)